### DISCUSSION TITLE
MDAnalysis is hiring! (Technical Writer)

### DISCUSSION THREAD

**Discussion by jennaswa** [MDAnalysis/mdanalysis#4703] [Announcements]
We are looking to hire a contract-based **Technical Writer** (fully remote; full- or part-time for 200 hours) to help reorganize and consolidate the MDAnalysis website and learning resources. See the attached expression of interest for full details, but in short the work will involve:

- Consolidating materials to remove duplication
- Identifying and updating outdated materials
- Differentiating between developer-focused and user-focused resources
- Integrating workshop materials and the MDAnalysis User Guide

If you're interested, please send your CV (including ≥2 references) and a cover letter illustrating how your experience and qualifications match the job description to [jobs@mdanalysis.org](mailto:jobs@mdanalysis.org) using the subject line: “MDAnalysis Technical Writer”. Cover letters should include links to online portfolios and profiles with demonstrations of work (e.g., GitHub profiles and repositories, personal websites, publications with data and code). 

[MDAnalysis - Expression of Interest - Technical Writer.pdf](https://github.com/user-attachments/files/16909723/MDAnalysis.-.Expression.of.Interest.-.Technical.Writer.pdf)

**Comment by Schefflera-Arboricola** [comment]
Hi, I'm interested. 

I documented a lot of the [NetworkX's backend dispatching](https://networkx.org/documentation/latest/reference/backends.html) while working on the [nx-parallel](https://github.com/networkx/nx-parallel) backend. I have started reading through the docs of MDAnalysis and I think it will take me some time to get familiar with the codebase and docs. I would really like to know if there is a recommended way I should read them or anything I should keep in mind.

Please LMK what are the next steps and if you have any open community meetings I could join to get involved in the community.

Thank you :)

**Comment by Asif-Iqbal-Bhatti** [comment]
I am intetested. How can i send you my CV.

**Comment by V-Alizade** [comment]
> We are looking to hire a contract-based **Technical Writer** (fully remote; full- or part-time for 200 hours) to help reorganize and consolidate the MDAnalysis website and learning resources. See the attached expression of interest for full details, but in short the work will involve:
> 
> - Consolidating materials to remove duplication
> - Identifying and updating outdated materials
> - Differentiating between developer-focused and user-focused resources
> - Integrating workshop materials and the MDAnalysis User Guide
> 
> Please get in touch here or with any of the @MDAnalysis/coredevs if you are interested or have any questions!
> 
> [MDAnalysis - Expression of Interest - Technical Writer.pdf](https://github.com/user-attachments/files/16909723/MDAnalysis.-.Expression.of.Interest.-.Technical.Writer.pdf)

Hi, I am interested too.

**Comment by jennaswa** [comment]
Thank you @Schefflera-Arboricola, @Asif-Iqbal-Bhatti and @V-Alizade for your interest. I have edited the job posting with more details on how you can apply as follows:

> If you're interested, please send your CV (including ≥2 references) and a cover letter illustrating how your experience and qualifications match the job description to [jobs@mdanalysis.org](mailto:jobs@mdanalysis.org) using the subject line: “MDAnalysis Technical Writer”. Cover letters should include links to online portfolios and profiles with demonstrations of work (e.g., GitHub profiles and repositories, personal websites, publications with data and code).

**Comment by amirhs1** [comment]
What is the deadline for the job post?

**Reply by jennaswa** [reply]
Applications will be reviewed on a rolling basis.
